common/src/autogluon/common/utils/utils.py

```diff
@@ -176,3 +177,14 @@ def hash_pandas_df(df: Optional[pd.DataFrame]) -> str:
     else:
         hashable_object = "0".encode("utf-8")
     return md5(hashable_object).hexdigest()
+
+
+def seed_everything(seed: int) -> None:
+    """Set random seeds for numpy and PyTorch."""
+    np.random.seed(seed)
+    try:
+        import torch
+
+        torch.manual_seed(seed)
+    except ImportError:
+        pass
```

Comment:
@yinweisu do we have this functionality anywhere else in `core` or `common`, or is it fine to add it here?

Reply:
I think it's fine to add it here